recipe/pkg-config

```diff
@@ -40,24 +40,32 @@ function _run_pkg_config() {
       SRLIBDIR64=
     fi
     if [[ ${CONDA_BUILD_PKG_CONFIG_LOG} == yes ]]; then
-      echo "Calling: PKG_CONFIG_LIBDIR=${PC_PATH_HOST}:$(${CC} -print-sysroot)/usr/share/pkgconfig${SRLIBDIR}${SRLIBDIR64} \
+      echo "Calling: PKG_CONFIG_LIBDIR=${PKG_CONFIG_LIBDIR}:${PC_PATH_HOST}:$(${CC} -print-sysroot)/usr/share/pkgconfig${SRLIBDIR}${SRLIBDIR64} \
         ${PC_PREFIX}/bin/pkg-config.bin --define-prefix --debug "$@" > /tmp/pkg-config-$$.log"
-      PKG_CONFIG_LIBDIR="${PC_PATH_HOST}":"$(${CC} -print-sysroot)/usr/share/pkgconfig${SRLIBDIR}${SRLIBDIR64}" \
+      PKG_CONFIG_LIBDIR="${PKG_CONFIG_LIBDIR}:${PC_PATH_HOST}":"$(${CC} -print-sysroot)/usr/share/pkgconfig${SRLIBDIR}${SRLIBDIR64}" \
         ${PC_PREFIX}/bin/pkg-config.bin --define-prefix --debug "$@" >> /tmp/pkg-config-$$.log 2>&1 || true
     fi
-    PKG_CONFIG_LIBDIR="${PC_PATH_HOST}":"$(${CC} -print-sysroot)/usr/share/pkgconfig${SRLIBDIR}${SRLIBDIR64}" \
+    PKG_CONFIG_LIBDIR="${PKG_CONFIG_LIBDIR}:${PC_PATH_HOST}":"$(${CC} -print-sysroot)/usr/share/pkgconfig${SRLIBDIR}${SRLIBDIR64}" \
       ${PC_PREFIX}/bin/pkg-config.bin --define-prefix "$@"
   else
     # We might need --define-prefix here sometimes, but would be better off ensuring /usr is replaced by ${prefix}
     # in all source .pc files in our packages. TODO :: Make conda-build / conda-verify detect this.
-    PC_PATH_SYSROOT=${PC_PATH_BUILD//$PC_PREFIX//usr$CONDA_BUILD_SYSROOT}
+    # try to determine additionnals /lib arch dir. On 64bits systems, it should be /lib and /lib64, on 32bits it should be /lib and /lib32
+    SRLIBDIR=":/usr/lib/pkgconfig:/usr/share/pkgconfig"
```

Comment:
@mingwandroid, what do you think about adding system paths here?

@StudioEtrange, these locations are not exhaustive. For eg on debian, you need `/usr/lib/<arch>`

Reply:
* Should i add /usr/lib/<arch> to this PR ?
* I was thinking it was the good place for tweak some system path (like in #30 ) because PKG_CONFIG_LIBDIR is used there

* If you consider that some system paths should not be added here , at least integrate the fix about PKG_CONFIG_LIBDIR so that we can set it by env var